### PR TITLE
RavenDB-13620 Moving disk size calculation to PAL

### PR DESCRIPTION
**Pull request by ayende**

### CODE REVIEW

File: src/Raven.Pal/pal.ver [hunk 1]
```diff
@@ -1 +1 @@
-42007
+42014
```
Comment (ppekrol) [inline src/Raven.Pal/pal.ver]: Can we bump only by 1?

Comment (ayende) [reply src/Raven.Pal/pal.ver]: Yes, auto generated

**Comment by ayende** [comment]
Wait, need to rebuilt the binaries, too

**Comment by ppekrol** [comment]
Can we squash everything to one commit to avoid pushing DLLs few times?

**Comment by ayende** [comment]
Yes, I'll rebase when done

**Review by aviviadi — APPROVED** [review]

**Pull request merged**